### DISCUSSION TITLE
showing wrong ip.

### DISCUSSION THREAD

**Discussion by portlek** [nuxt/nuxt#17455] [Questions] [answered]
there is no issue about running nuxt on localhost, it's just showing wrong ip on the console.

it shows:
`Listening on http://0.0.0.0:3000/`

supposed to be:
`Listening on http://localhost:3000/`

is this related to nuxt?

using
`"nuxt": "3.0.0-rc.1"`

**Comment by danielroe** [comment, marked as answer]
We now default production host to `0.0.0.0` (which is your public interface) - see https://github.com/unjs/nitro/commit/b7c719349a0ee399a525937abb5905e50b0351f2.

**Reply by StanleyMasinde** [reply]
I think the default would be 127.0.0.1 instead of the public 0.0.0.0 which is usually used for containerised apps.

**Comment by Johannes-Schiel** [comment]
I have the same problem and in my system NITRO_HOST and NITRO_PORT are ignored how do you have your nuxt 3 config to use Listening on http://localhost:3000/ ? @portlek

**Reply by danielroe** [reply]
Would you provide more information on how you are running the built Nitro server?

**Reply by Johannes-Schiel** [reply]
plain default just a new nuxtjs application when i build the app and run that with node .output/server/index.mjs its starts on http://0.0.0.0:3000 but i cant use port 3000 because its used for another app.

**Reply by danielroe** [reply]
You have to set the environment variable. For example, `PORT=4000 node .output/server/index.mjs`.

**Reply by Johannes-Schiel** [reply]
Thats what i did, but its not realy working still starts with http//0.0.0.0:3000 if i write PORT=4000 in .env should that work?

**Reply by danielroe** [reply]
No, your .env file is not read in production. You need to set the environment variables yourself. There are many different ways to do this and it depends on your deployment target.
